src/main/java/use_case/freeplay/hit/HitUseCaseInteractor.java

```diff
@@ -32,17 +32,17 @@ public void execute() {
         UserPlayer player = gameDAO.getPlayer();
         ArrayList<Card> playerHand = player.getHand();
 
-        //if the player can take their turn
-        if (canHit(getHandValue(playerHand))) {
-            //hit branch
-            //add card to player hand
-            playerHand.add(freePlayDataAccessObject.getCard(gameDAO.getDeckID()));
-            player.setHand(playerHand);
-            //update player
-            gameDAO.setPlayer(player);
+        //add card to player hand
+        playerHand.add(freePlayDataAccessObject.getCard(gameDAO.getDeckID()));
+        player.setHand(playerHand);
+        //update player
+        gameDAO.setPlayer(player);
 
-            //output data and presenter
-            HitOutputData outputData = new HitOutputData(makeImages(playerHand));
+        //checking if they've bust or won
+        if (canHit(getHandValue(playerHand))) {
+            //successful hit, success view
+            HitOutputData outputData = new HitOutputData(makeImages(playerHand),
+                    makeImages(gameDAO.getDealer().getHand()));
```

Comment:
It seems like the card is now added before checking if they've bust or won. I think this is a good decision as they should be able to hit as many times as they press the button